AsyncDisplayKit/Details/Transactions/_ASAsyncTransaction.mm

```diff
@@ -8,22 +8,24 @@
 //  of patent rights can be found in the PATENTS file in the same directory.
 //
 
+#import <UIKit/UIApplication.h>
```

Comment:
@Adlai-Holler I may have missed it, but what adds this requirement?  I'd prefer to not have a dependency on UIApplication at this level.